demo/admin/src/pages/Page.tsx

```diff
@@ -4,7 +4,8 @@ import { DocumentInterface, rewriteInternalLinks } from "@comet/cms-admin";
 import { PageTreePage } from "@comet/cms-admin/lib/pages/pageTree/usePageTree";
 import { Chip } from "@mui/material";
 import { SeoBlock } from "@src/common/blocks/SeoBlock";
-import { GQLPage, GQLPageInput, GQLPageTreeNodeAdditionalFieldsFragment } from "@src/graphql.generated";
+import { GQLPageTreeNodeAdditionalFieldsFragment } from "@src/common/EditPageNode";
```

Comment:
this import was invalid before:

PageTreeNodeAdditionalFields Fragment is defined in EditPageNode.tsx and additionally used in Page.tsx. This is (strictly speaking) invalid, as the fragment is a private part of EditPageNode.tsx. Easy fix: make it public by exporting it, then we can import it here.